index.js

```diff
@@ -1,88 +1,131 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+let counter = 1
+counter = 2
+
+console.log("%cQuestion 1", "color: red") 
+
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
 // console.log("%cQuestion 2", "color: red") 
+ const name = "Yehudis"
 
 // console.log(name)
-// => "Raffy"
+// // => "Raffy"
 // name = "Not Raffy" 
-// => TypeError
+// // => TypeError
 // console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
 // console.log("%cQuestion 3", "color: red") 
-
-// console.log(drinkWater(12))
+function drinkWater(currentThirstLevel) {
+  console.log('Man I sure am thirsty')
+  console.log('Ahh that hits the spot')
+  return currentThirstLevel - 1
+}
+ console.log(drinkWater(12))
```

Comment:
Great work! In order to keep consistent with the Ruby example, you would write line 35 as

`return currentThirstLevel -= 1`